superset/databases/schemas.py

```diff
@@ -397,6 +397,7 @@ class Meta:  # pylint: disable=too-few-public-methods
     )
     is_managed_externally = fields.Boolean(allow_none=True, default=False)
     external_url = fields.String(allow_none=True)
+    uuid = fields.String(allow_none=True)
```

Comment:
should we default to some generated uuid if not provided? I think making it allow_none=False will be more helpful.

Reply:
we might need some migration to backfill though

Reply:
Right, this allows the user to pass `None` for the UUID, and in that case a random one will be generated.

Reply:
@zephyring I changed it so that it's not required, but if present it can't be `None`, which is what we want.